### ISSUE TITLE
Darken Background of Note Tags To Increase Contrast

### ISSUE THREAD

**Issue by SylvesterWilmott** [Automattic/simplenote-android#828]
Currently, in dark mode, we use `gray_60` for the tag/chip backgrounds within the notes view. Let's change this to `gray_70` to increase contrast with the text.

`gray_80` should be used for the pressed state.
